check_line_length.py

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/python
```

Comment:
@waleko `flake8` works on python files.
`check_line_length.py` is for RST files, I think that's what @ppabcd is pointing at, let me know if I have misunderstood something!

Reply:
I returned `check_line_length.py`